Assets/SteamVR_Unity_Toolkit/Scripts/Helper/DeviceFinder.cs

```diff
@@ -35,9 +35,10 @@ public static SteamVR_TrackedObject TrackedObjectByIndex(uint controllerIndex)
         public static Transform HeadsetTransform()
         {
 #if (UNITY_5_4_OR_NEWER)
-        return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
-#endif
+            return GameObject.FindObjectOfType<SteamVR_Camera>().GetComponent<Transform>();
+#else
```

Comment:
No, you're correct, if the warning is throwing in Unity 5.4 then your fix is the right way to go.

However, I'd push just this fix on it's own in it's own Pull Request. As the other changes in this PR may well be superseded by the other pull request I've referenced.